frontends/web/src/components/layout/header.module.css

```diff
@@ -21,12 +21,16 @@
     align-items: center;
     flex: none;
     margin: 0 auto;
+    flex-wrap: wrap;
+    padding: var(--space-default);
 }
```

Comment:
please sort all properties alphabetically, so that it's clearly visible that `padding` is defined twice.